weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallTraceView.tsx

```diff
@@ -249,6 +249,9 @@ const RE_TRAILING_INT = /\d+$/;
 // Sorting evaluation calls by dataset row.
 // Because of async they may have been processed in a different order.
 const getCallSortExampleRow = (call: CallSchema): number => {
+  if (!call.rawSpan.inputs?._keys) {
```

Comment:
`_keys` is not right here - not sure why this is a check?

Reply:
?? dev tracks I think. Sloppy